src/pyvoices/commands/chunk.py

```diff
@@ -32,7 +32,7 @@ def run(args):
     audio = get_mono_channel(d.audio)  # FIXME: pass in channel number from args!
     intervalByConcept = {i.label: i for i in d.praat_textgrid.tierDict[args.concept_tier].entryList}
     count = 0
-    for t in d.iter_transcriptions():
+    for t in d.iter_transcriptions(args.concept_tier, args.concept_column):
```

Comment:
Please pass keyword arguments always as such. Relying on the position is dangerous.